vantage6/cli/node.py

```diff
@@ -309,13 +309,16 @@ def cli_node_start(name, config, environment, system_folders, image, keep,
     info("Creating Docker data volume")
     data_volume = docker_client.volumes.create(
         f"{ctx.docker_container_name}-vol")
+    vpn_volume = docker_client.volumes.create(
+        f"{ctx.docker_container_name}-vpn-vol")
```

Comment:
It would be nicer to use the new `ctx.docker_vpn_volume_name` property here.

Reply:
Done